docs/src/pages/discover-more/team/Team.js

```diff
@@ -31,59 +24,85 @@ const members = [
     flag: 'Core team',
     city: 'London, UK',
   },
-  {
-    name: 'Kevin Ross',
-    github: 'rosskevin',
-    twitter: 'rosskevin',
-    flag: 'Core team',
-    city: 'Franklin, Tennessee, US',
-  },
   {
     name: 'Sebastian Silbermann',
     github: 'eps1lon',
     twitter: 'sebsilbermann',
     flag: 'Core team',
     city: 'Dresden, Germany',
   },
+  {
+    name: 'Josh Wooding',
+    github: 'joshwooding',
+    flag: 'Core team',
+    city: 'UK',
+  },
+  {
+    name: 'Maik Marschner',
+    github: 'leMaik',
+    twitter: 'leMaikOfficial',
+    flag: 'Core Team',
+    city: 'Hannover, Germany',
+  },
+  {
+    name: 'Tom Crockett',
+    github: 'pelotom',
+    twitter: 'pelotom',
+    flag: 'Core Team',
+    city: 'Los Angeles, California, US',
+  },
+];
+
+const emeriti = [
+  {
+    name: 'Hai Nguyen',
+    github: 'hai-cea',
+    twitter: 'haicea',
+    flag: 'v0.x creator',
+    city: 'Dallas, Texas, US',
+  },
   {
     name: 'Nathan Marks',
     github: 'nathanmarks',
     flag: 'v1.x co-creator',
     city: 'Toronto, ON',
   },
+  {
+    name: 'Kevin Ross',
+    github: 'rosskevin',
+    twitter: 'rosskevin',
+    flag: 'Core team',
+    city: 'Franklin, Tennessee, US',
+  },
   {
     name: 'Sebastian Sebald',
     github: 'sebald',
     twitter: 'sebastiansebald',
-    flag: 'Community partner, TypeScript',
+    flag: 'Core Team',
     city: 'Freiburg, Germany',
   },
   {
-    name: 'Maik Marschner',
-    github: 'leMaik',
-    twitter: 'leMaikOfficial',
-    flag: 'Community partner',
-    city: 'Hannover, Germany',
+    name: 'Ken Gregory',
+    github: 'kgregory',
+    flag: 'Core Team',
+    city: 'New Jersey, US',
   },
+];
+
+const partners = [
   {
     name: 'Oleg Slobodskoi',
     github: 'kof',
     twitter: 'oleg008',
-    flag: 'Community partner, JSS',
+    flag: 'JSS',
     city: 'Berlin, Germany',
   },
   {
-    name: 'Ken Gregory',
-    github: 'kgregory',
-    flag: 'Community partner',
-    city: 'New Jersey, US',
-  },
-  {
-    name: 'Tom Crockett',
-    github: 'pelotom',
-    twitter: 'pelotom',
-    flag: 'Community partner',
-    city: 'Los Angeles, California, US',
+    name: 'dmtrKovalenko',
```

Comment:
I am really glad to see myself here listed in the team. But can I ask to change the name to `Dmitriy Kovalenko` thanks :)

Reply:
```suggestion
    name: 'Dmitriy Kovalenko',
```

Reply:
Oh I'm sorry! Good catch :). Thank you for your work <3